generators/liquibase/generator.ts

```diff
@@ -27,6 +27,7 @@ import {
   postPrepareEntity,
   prepareRelationshipForLiquibase,
   liquibaseComment,
+  htmlEncode,
 } from './support/index.js';
```

Comment:
```suggestion
} from './support/index.js';
import { escape } from 'lodash-es';
```

Reply:
ah, ok, I thought, the custom implementation was there for a reason ;-) Will change